src/pages/export-page/ExportPage.test.tsx

```diff
@@ -1,22 +1,34 @@
 import "@testing-library/jest-dom";
 import { act, render, screen } from "@testing-library/react";
 
+import {
+  DEFAULT_DEVELOPMENT_PLAN,
+  TimetableEventKey,
+  getFormattedDate,
+} from "@lib/constants";
 import {
   resolveDevelopmentPlanCSV,
   resolveTimetableEventsCSV,
 } from "@lib/utils/timetable";
 import { PlanViewer } from "@lib/timetable-visualisation/PlanViewer";
-import { FormProvider } from "../../context/FormContext";
+import { FormContextValues, FormProvider } from "../../context/FormContext";
+import { Journey } from "../../routes/routes";
 import { ExportPage } from "./ExportPage";
+import { DevelopmentPlanTimetable } from "@lib/types/timetable";
 
 jest.mock("@lib/utils/timetable");
 jest.mock("@lib/timetable-visualisation/PlanViewer");
 
 jest.mock("react-router-dom");
 
-const renderPage = () =>
+jest.mock("@lib/constants", () => ({
+  ...jest.requireActual("@lib/constants"),
+  getFormattedDate: jest.fn(),
+}));
+
+const renderPage = (contextValues?: Partial<FormContextValues>) =>
   render(
-    <FormProvider>
+    <FormProvider contextValues={contextValues}>
```

Comment:
can we mock the provider?

Reply:
actually I think for the scope of this page we can just give a mock value to the userFlow we get from useFormContext() hook

Reply:
Discussed offline, now mocking the hook instead of using the provider